app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php

```diff
@@ -292,7 +292,7 @@ protected function _getDefaultBackend() {
             'first_byte_timeout'    => $timeout . 's',
             'between_bytes_timeout' => $timeout . 's',
         );
-        if ( Mage::getStoreConfigFlag( 'turpentine_vcl/backend/load_balancing' ) ) {
+        if ( Mage::getStoreConfigFlag( 'turpentine_vcl/backend/load_balancing' ) != 'no' ) {
```

Comment:
This seems to be the only relevant code change, the other ones seems to be indentation and style. Can you confirm @BarryCarlyon ?